ember-model.js

```diff
@@ -2111,16 +2111,16 @@ Ember.Model.Store = Ember.Object.extend({
 Ember.onLoad('Ember.Application', function(Application) {
 
   Application.initializer({
-    name: "store",
+    name: "emstore",
 
     initialize: function() {
       var application = arguments[1] || arguments[0];
-      var store = application.Store || Ember.Model.Store;
-      application.register('store:application', store);
-      application.register('store:main', store);
+      var emstore = application.Store || Ember.Model.Store;
+      application.register('emstore:application', emstore);
+      application.register('emstore:main', emstore);
```

Comment:
Not to discourage this change just for information purposes, as of 2.0.0 Ember Data no longer registers anything on `store:main` or `store:application` instead Ember Data registers its store to `service:store`.